### PR TITLE
copy changes on the deployment dashboard page in tyk cloud

### PR DESCRIPTION
**Pull request by ifrim**
### For internal users - please add a Jira DX PR ticket to the subject!
tyk cloud ticket - https://tyktech.atlassian.net/browse/TT-9533

---
### Preview Link
https://deploy-preview-2990--tyk-docs.netlify.app/docs/nightly/tyk-cloud/environments-deployments/hybrid-gateways/

### Description
There are some copy changes in Tyk Cloud on the deployment dashboard page.
This PR replaces **API Manager Dashboard** with **Tyk Dashboard** in the docs page `/tyk-cloud/environments-deployments/hybrid-gateways/`.
It also updates two screenshots of Tyk Cloud in the same docs page to show the latest changes.

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

### CODE REVIEW

**Review by SonjaChevre — APPROVED** [review]
looks good, thanks!

File: tyk-docs/content/tyk-cloud/environments-deployments/hybrid-gateways.md [hunk 1]
```diff
@@ -206,9 +206,9 @@ Follow the notes from the installation output to get connection details and pass
   export REDIS_PASSWORD=$(kubectl get secret --namespace tyk tyk-redis -o jsonpath="{.data.redis-password}" | base64 --decode)
 ```
 
-You need to modify the following values in your custom `values.yaml` file: 
+You need to modify the following values in your custom `values.yaml` file:
 
-* `redis.addrs`: the name of the redis instance including the port as set by Bitnami `tyk-redis-master.tyk.svc.cluster.local:6379` 
+* `redis.addrs`: the name of the redis instance including the port as set by Bitnami `tyk-redis-master.tyk.svc.cluster.local:6379`
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-cloud/environments-deployments/hybrid-gateways.md]:
```suggestion
* `redis.addrs`: the name of the Redis instance including the port as set by Bitnami `tyk-redis-master.tyk.svc.cluster.local:6379`
```

File: tyk-docs/content/tyk-cloud/environments-deployments/hybrid-gateways.md [hunk 2]
```diff
@@ -18,30 +18,30 @@ This page describes the deployment of hybrid data planes and how to connect them
 ## Pre-requisites
 
 * Tyk Cloud Account, register here if you don't have one yet: {{< button_left href="https://tyk.io/sign-up/#cloud" color="green" content="free trial" >}}
-* A redis instance for each data plane, used as temporay storage for distributed rate limiting, token storage and analytics. You will find instructions for a simple Redis installation in the steps below. 
+* A redis instance for each data plane, used as temporay storage for distributed rate limiting, token storage and analytics. You will find instructions for a simple Redis installation in the steps below.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-cloud/environments-deployments/hybrid-gateways.md]:
```suggestion
* A Redis instance for each data plane, used as temporay storage for distributed rate limiting, token storage and analytics. You will find instructions for a simple Redis installation in the steps below.
```

**Comment by dcs3spp** [comment]
@letzja are terminology changes ok?

**Review by letzya — APPROVED** [review]

**Comment by letzya** [comment]
@dcs3spp approved. We can merge this

**Pull request merged**